packages/app/src/runner/unifiedRunner.ts

```diff
@@ -35,6 +35,10 @@ export function useUnifiedRunner () {
           return
         }
 
+        if (queryFile === '__all') {
```

Comment:
Should we use the constant 

```suggestion
        if (queryFile === RUN_ALL_SPEC['fileName']) {
```

Reply:
Done in [b0ba5c2](https://github.com/cypress-io/cypress/pull/24683/commits/b0ba5c2b0e86e79183a9715b58e6fa22c9aa703a)